packages/rx_bloc_generator/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [5.2.0] - July 27, 2022
+* Update dependencies
+* [BREAKING CHANGE] Update analyzer to ">=3.0.0 <=5.0.0"
```

Comment:
According to the semantic versioning if there is a breaking change the major version should be increased. If that's the case, please bump the package version to 6.0.0

Reply:
I wasn't aware of such convention. Will change it to `6..0.0`.

Reply:
Please add a bullet point describing this new feature - https://github.com/Prime-Holding/rx_bloc/pull/197